src/stores/RootStore.ts

```diff
@@ -106,10 +109,54 @@ export default class RootStore {
 
 	private parseUrlState = (): WorkspacesUrlState | null => {
 		try {
+			if (window.location.search.split('&').length > 1) {
+				throw new Error('Only one query parameter expected.');
+			}
 			const searchParams = new URLSearchParams(window.location.search);
 			const workspacesUrlState = searchParams.get('workspaces');
-			const parsedState = workspacesUrlState ? JSON.parse(window.atob(workspacesUrlState)) : null;
-			return parsedState;
+			const timestamp = searchParams.get('timestamp');
+			const eventId = searchParams.get('eventId');
+			const messageId = searchParams.get('messageId');
+			if (workspacesUrlState) {
+				return JSON.parse(window.atob(workspacesUrlState));
+			}
+			const interval = intervalOptions[0];
+			const defaultUrlState = {
+				interval,
+				timeRange: getRangeFromTimestamp(Date.now(), interval),
+				layout: defaultPanelsLayout,
+			};
+			if (timestamp) {
```

Comment:
We should be able to initialize workspace without these if-statements since initial status should accept empty values
something like that
`
const state: WorkspaceUrlState = {
  interval,
  layout: defaultPanelsLayout,
  events: eventId || {},
  messages: messageId || {},
  timeRange: timestamp ? getRangeFromTimestamp(+timestamp, interval) : null,
};
`